tempoiq/client.py

```diff
@@ -82,9 +83,13 @@ class Client(object):
     create_encoder = CreateEncoder()
     read_encoder = ReadEncoder()
 
-    def __init__(self, endpoint):
+    def __init__(self, endpoint, read_version='v2'):
         self.endpoint = endpoint
         self.monitoring_client = MonitoringClient(self.endpoint)
+        global DATAPOINT_ACCEPT_TYPE
+        DATAPOINT_ACCEPT_TYPE = media_type('datapoint-collection',
```

Comment:
I think this screws up if you create multiple clients with different read versions since they will both reference the same global.

I think we should move the *_ACCEPT_TYPE and *_CONTENT_TYPE into the client local scoping